solidity/test/integration/utils/ecdsa-wallet-registry.ts

```diff
@@ -0,0 +1,269 @@
+// TODO: Utils in this file are pulled from @keep-network/ecdsa test utils.
+// We should consider exposing them in @keep-network/ecdsa for an external usage.
+
+/* eslint-disable no-await-in-loop */
+import { deployments, ethers, helpers } from "hardhat"
+
+import type { SignerWithAddress } from "@nomiclabs/hardhat-ethers/signers"
+import type {
+  ContractTransaction,
+  BytesLike,
+  BigNumber,
+  BigNumberish,
+  Contract,
+  Signer,
+} from "ethers"
+import type { WalletRegistry, SortitionPool } from "../../../typechain"
+
+// Number of members in the ECDSA Wallet.
+const WALLET_SIZE = 100
+
+export async function registerOperator(
+  walletRegistry: WalletRegistry,
+  sortitionPool: SortitionPool,
+  stakingProvider: Signer,
+  operator: Signer
+): Promise<number> {
+  await walletRegistry
+    .connect(stakingProvider)
+    .registerOperator(await operator.getAddress())
+
+  await walletRegistry.connect(operator).joinSortitionPool()
+
+  const operatorID = await sortitionPool.getOperatorID(
+    await operator.getAddress()
+  )
+
+  return operatorID
+}
+
+export async function produceEcdsaDkgResult(
```

Comment:
Maybe `performDkg`? It's not only about producing a result that is later submitted by some other function but also about selecting group members, submitting and approving that result. It's a full DKG path.

Reply:
Good suggestion. I'd name it `performEcdsaDkg` to be more precise, as we will have a similar for the random beacon.

Reply:
Sure, `performEcdsaDkg` works.

Reply:
006efd4e6d1316b21b322933c16fbff7be5b28e0